package.json

```diff
@@ -68,11 +68,11 @@
     "server": "npm start",
     "snyk": "snyk",
     "snyk-protect": "snyk protect",
-    "start": "npm run babel:build && http-server",
+    "start": "npm run build:components && http-server",
     "test:build": "mocha ./tests/test-build.js",
     "test:headless": "node ./tests/headless.js",
     "test:karma": "npm run karma",
-    "prepare": "npm run snyk-protect",
+    "prepare": "npm run snyk-protect && npm run copydeps && npm run build:components",
```

Comment:
I hope the snyk bot will not replace this line 😀

Reply:
`build:components` includes `copydeps` so we can omit the copydeps part.

Reply:
> I hope the snyk bot will not replace this line 

I don't think it will 🤞... I think it's just a standard npm thing, so Snyk should not override it.